### PR TITLE
Xode PR 7_3_1

### PR DESCRIPTION
**Pull request by Xode672**

### CODE REVIEW

**Comment by rschlaikjer** [comment]
This isn't using the most recent script as a base. Always pull main before starting a file.

File: script/Arcueid/Day 7/07_00_ARC07_3_1.txt [hunk 1]
```diff
@@ -48,7 +48,7 @@ Don't humans usually get sleepy at this time of night?"
 [sha:ccefe4a1e84c3da14c1fdd24254a5b940a5f2b20]{
 -- Page 502, Offset 17819.
 -- 「ふーん。夢は見ないの？」
-"Hmm. Do you not see any dreams?"
+"Hmm. You don't get any dreams?"
```
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3_1.txt:51]: 'getting' dreams feels weird, maybe rephrase to use dream as verb?
> Hmm. Do you not dream or something?

File: script/Arcueid/Day 7/07_00_ARC07_3_1.txt [hunk 2]
```diff
@@ -110,7 +110,7 @@ Geez, what kind of person do you take me for?! Are you misunderstanding somethin
 [sha:cf209a990933d095c5dc3326c5e9107a9aa36d8b]{
 -- Page 507, Offset 17831.
 -- 「大丈夫、そこは完璧、ブレてない。アルクェイドの事は、手加減のできないやりすぎ吸血鬼だと思ってる」
-"Don't worry, I know exactly who you are. You're a vampire who goes too far and doesn't know how to hold back."
+"Don't worry, I know exactly what to take you for. A vampire who goes too far and doesn't know how to hold back."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3_1.txt:113]: Not sure that this sees common usage, but I do appreciate the mirroring. On the fence, would be interested in @Avuxo's opinion

Comment (Avuxo) [reply script/Arcueid/Day 7/07_00_ARC07_3_1.txt:113]: this isn't necessarily a common usage but I think I like it tbqh. I say let's go with it.

File: script/Arcueid/Day 7/07_00_ARC07_3_1.txt (hunk 1 above)
Comment (Avuxo) [reply script/Arcueid/Day 7/07_00_ARC07_3_1.txt:51]: I disagree, I think this is a pretty natural expression

**Review by Avuxo — APPROVED** [review]

**Review by rschlaikjer — APPROVED** [review]
